src/ssda/util/salt_observation.py

```diff
@@ -55,7 +55,10 @@ def create_reduced_path(path: Path) -> Optional[Path]:
 
             for _path in reduced_paths:
```

Comment:
It might be a bit unfortunate to have both a variable _path and a variable path -- these names are quite similar... Maybe consider renaming path to raw_path? (Assuming it is the path of the raw file.)